sesotho/voc.txt

```diff
@@ -0,0 +1,77 @@
+motho
+batho
+moruti
+baruti
+mosadi
+basadi
+ngwana
+bana
+morena
+marena
+lefapha
+mafapha
+lerato
+marato
+sebaka
+libaka
+setso
+meetlo
+botle
+molimo
+balimo
+maikutlo
+boikutlo
+motse
+metse
+toropo
+toropong
+ntlo
+matlo
+mosebetse
+mesebetse
+meetlo
+moetlo
+mosuwe
+basuwe
+rutile
+rutisa
+rutileng
+rata
+ratile
+ratisa
+ratisang
+bapala
+bapalang
+bapilwe
+bapisa
+bapetse
+phetse
+phetseng
+tsamaya
+tsamaile
+tsamaisang
+etsa
+etsile
+etsisa
+etsang
+sheba
+shebile
+shebisang
+bolela
+boletse
+bua
+buile
+reka
+rekile
+rekisa
+Sebata
+pholosa
+Rongoa
+Mothusi
```

Comment:
I think if you lower-case the capitalised words in this file that'll fix the CI failures (it's arguably a bug that e.g. the Ada version of stemwords doesn't appear to, but I'm not sure I know enough Ada to fix that).